ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -157,7 +157,7 @@
 		97C146E61CF9000F007C117D /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
-				LastUpgradeCheck = 1430;
+				LastUpgradeCheck = 1510;
```

Comment:
This change (and a similar one in other files) looks like it comes from upgrading your Xcode, rather than from any change in Flutter.